react/components/molecules/image-carrousel/image-carrousel.js

```diff
@@ -0,0 +1,187 @@
+        this.setState(
+            { currentPagePosition: currentPagePosition, selectedImageData: imageIndex },
+            () => {
+                this.setVisibility(true);
+            }
+        );
+    }
+
+    closeLigthBox() {
+        this.setVisibility(false);
+    }
+
+    onScroll = event => {
+        const scroll = event.nativeEvent.contentOffset.x;
+
+        const currentPage = Math.round(scroll / this.scrollCoeficient);
+        this.setState({ selectedImageData: currentPage });
+    };
+
+    onBackButtonPress = () => {
+        this.closeLigthBox();
+    };
+
+    onClosePress = () => {
+        this.closeLigthBox();
+    };
+
+    onLightboxPress = () => {
+        this.setVisibility(true);
+    };
+
+    _imageSource = image => {
+        return image.uri ? { uri: image.uri } : image.src;
+    };
+
+    _imageFullscreenStyle = image => {
+        return [
+            styles.imageFullscreen,
+            {
+                resizeMode: this.props.resizeModeFullScreen,
+                width: this.scrollCoeficient
+            },
+            image?.style
+        ];
+    };
+
+    render() {
+        return (
+            <View style={styles.container}>
+                <Modal
+                    animationType="fade"
+                    transparent={false}
+                    visible={this.state.visible}
+                    onRequestClose={this.onBackButtonPress}
+                >
+                    <View style={styles.fullscreenContainer}>
+                        <ScrollView
+                            contentOffset={{ x: this.state.currentPagePosition, y: 0 }}
+                            style={{ flex: 1 }}
+                            contentContainerStyle={{ flexGrow: 1 }}
+                            horizontal={true}
+                            pagingEnabled={true}
+                            showsHorizontalScrollIndicator={true}
+                            onScroll={this.onScroll}
+                        >
+                            {this.props.images.map((image, index) => (
+                                <Image
+                                    key={index}
+                                    style={this._imageFullscreenStyle(image)}
+                                    source={this._imageSource(image)}
+                                />
+                            ))}
+                        </ScrollView>
+                        <ButtonIcon
+                            icon={"close"}
+                            onPress={this.onClosePress}
+                            style={styles.buttonClose}
+                            iconStrokeWidth={2}
+                            size={isTabletSize() ? 52 : 34}
+                            iconHeight={isTabletSize() ? 34 : 22}
+                            iconWidth={isTabletSize() ? 34 : 22}
+                            backgroundColor={"#000000"}
+                            iconStrokeColor={"#ffffff"}
+                        />

```

Comment:
Please re-order the  props following our standard:
```suggestion
<ButtonIcon
    style={styles.buttonClose}
    icon={"close"}
    iconStrokeWidth={2}
    size={isTabletSize() ? 52 : 34}
    iconHeight={isTabletSize() ? 34 : 22}
    iconWidth={isTabletSize() ? 34 : 22}
    backgroundColor={"#000000"}
    iconStrokeColor={"#ffffff"}
    onPress={this.onClosePress}
/>

```